HW1/P2/P2b.py

```diff
@@ -1,3 +1,20 @@
 from P2 import *
 
 # Your code here
+
+# part 2b
+# make pixels
+i = sc.parallelize(range(2000))
+j = sc.parallelize(range(2000))
```

Comment:
Can call sc.range() instead of creating the ranges in python and then parallelizing them.